packages/synapse-constants/constants/chains/index.ts

```diff
@@ -1,34 +1,38 @@
+import { Chain } from '../types'
 import * as all from './master'
-import { Chain } from '../../utils/types/index'
+
+export * from './master'
 
 export type ChainsByChainID = {
   [cID: number]: Chain
 }
 export const sortChains = (chains: Chain[]) =>
-  Object.values(chains).sort((a, b) => b.priorityRank - a.priorityRank)
+  Object.values(chains).sort(
+    (a: Chain, b: Chain) => (b.priorityRank ?? 0) - (a.priorityRank ?? 0)
+  )
 
-export const CHAINS_ARR = Object.values(all).sort(
-  (a, b) => b.priorityRank - a.priorityRank
-)
+export const CHAINS_ARR = Object.values(all)
+  .filter((item): item is Chain => typeof item !== 'number')
+  .sort((a: Chain, b: Chain) => (b.priorityRank ?? 0) - (a.priorityRank ?? 0))
 
 const getChainEnumById = () => {
   const outObj: Record<number, string> = {}
-  CHAINS_ARR.map((chain) => {
+  CHAINS_ARR.map((chain: any) => {
     outObj[chain.id] = chain.codeName
   })
   return outObj
 }
 
 const getids = () => {
-  const outObj = {}
-  CHAINS_ARR.map((chain) => {
+  const outObj: { [key: string]: any } = {}
+  CHAINS_ARR.map((chain: any) => {
```

Comment:
I think we can pass in Chain type here right?

CHAINS_ARR.map((chain: Chain)) => ....)

Reply:
typed